pkgs/servers/etebase/default.nix

```
@@ -24,7 +25,8 @@ buildPythonPackage rec {
     pynacl
     redis
     typing-extensions
-  ] ++ lib.optional withLdap python-ldap;
+  ] ++ lib.optional withLdap python-ldap
+  ++ lib.optional withPostgres psycopg2;
```

```suggestion
    ++ lib.optional withPostgres psycopg2;
```

Usually the indentation is so the operators align.